extensions/wikia/ArticleVideo/scripts/video-feedback.js

```diff
@@ -0,0 +1,47 @@
+define('showVideoFeedbackBox', ['wikia.window', 'wikia.tracker'], function (window, tracker) {
```

Comment:
We have a lot of modules, let's use a namespace here, e.g. `wikia.articleVideo.showVideoFeedbackBox` (using https://github.com/Wikia/app/blob/dev/extensions/wikia/WikiaMaps/js/WikiaMapsPoi.js#L2 as a good example).